### ISSUE TITLE
Names of kit items are lost on save

### ISSUE THREAD

**Issue by y6nH** [Yoshimi/yoshimi#118]
I'm using Yoshimi 1.7.3 on Manjaro.

I'm putting together a drum kit. However, the names I enter in the Kit Edit view are not saved, so when I save and reopen the patch, all the kit element name fields are blank.

Looking inside the exported .xiy and .xiz files, the name elements are empty, like this: 

```xml
<INSTRUMENT_KIT_ITEM id="1">
<par_bool name="enabled" value="yes" />
<string name="name"></string>
<par_bool name="muted" value="no" />
...
```

Any ideas? Thanks!

**Comment by abrolag** [comment]
On Sat, 14 Nov 2020 15:21:17 -0800
Hugh <notifications@github.com> wrote:

>I'm using Yoshimi 1.7.3 on Manjaro.
>
>I'm putting together a drum kit. However, the names I enter in the Kit Edit view are not saved, so when I save and reopen the patch, all the kit element name fields are blank.
>
>Looking inside the exported .xiy and .xiz files, the name elements are empty, like this: 

This is a bug in the GUI :(

CLI entries work correctly, and the GUI sees them correctly too.
So if you don't mind a bit of messing about with the CLI you can set them
there.

Assuming part one, from a terminal ~$ prompt:

yoshimi
	click again in the terminal window (the GUI will have grabbed it)
s p mult
	will get you to part 1, kit 1 multi mode (you may prefer single)
s na kick
	will set item 1 name to kick
s 2
	moves to item 2
s on
	enables it (if it wasn't already)
s na snare
	set item 2 name to snare

rinse and repeat!


I'll put the fix into github, but it will be a while before I can get a bugfix
release out... and then for the distros to catch up.

-- 
Will J Godfrey
http://www.musically.me.uk
Say you have a poem and I have a tune.
Exchange them and we can both have a poem, a tune, and a song.

**Comment by y6nH** [comment]
Thanks for the quick response. I've never used the CLI, but I'll try it out.

**Comment by abrolag** [comment]
On Sun, 15 Nov 2020 03:00:00 -0800
Hugh <notifications@github.com> wrote:

>Thanks for the quick response. I've never used the CLI, but I'll try it out. 
>
No Prob.

It can do everything the the CLI can :)

I forgot to mention, you either need the 'Reports' window open, or in 'Settings'
set it to 'stdout' so you can see it is actually doing what you tell it!

-- 
Will J Godfrey
http://www.musically.me.uk
Say you have a poem and I have a tune.
Exchange them and we can both have a poem, a tune, and a song.

**Comment by abrolag** [comment]
Just an update on this. A maintenance release, Yoshimi V 1.7.4 will be out in early January (all being well) and this will be fixed in  it. it's then just a matter of the distros getting on-board.

**Comment by abrolag** [comment]
On Sun, 15 Nov 2020 03:00:00 -0800
Hugh <notifications@github.com> wrote:

>Thanks for the quick response. I've never used the CLI, but I'll try it out. 
>

This should now be fixed in the current master, and in the 1.7.4 release.
It might take a while for the distros to pick it up.

-- 
Will J Godfrey
http://www.musically.me.uk
http://yoshimi.github.io
Say you have a poem and I have a tune.
Exchange them and we can both have a poem, a tune, and a song.

**Comment by abrolag** [comment]
Fixed in V 1.7.4
